### PR TITLE
add Antithesis as a member

### PR DESCRIPTION
**Pull request by guergabo**
The blog post and links to JSON file will go live Sep 16. 

Branding materials: https://www.antithesis.com/brand/

### CODE REVIEW

**Comment by vladh** [comment]
Thank you! We’ll look out for the JSON file and review when it’s available. Excited to have you on board! 🐛

**Review by vladh — CHANGES_REQUESTED** [review]
Could we have `urlLearnMore` point to the blog post instead of the homepage? Just because the `urlLearnMore` page should say something about the member's contributions to Open Source.

**Comment by vladh** [comment]
I left one small comment, but this looks good to me otherwise! I'll wait for a second review before merging.

**Comment by chadwhitacre** [comment]
Everyone else has prepended instead of appended. Do we care?

**Comment by vladh** [comment]
> Everyone else has prepended instead of appended. Do we care?

I don't think it matters — we can just sort them alphabetically later. 😛

**Comment by chadwhitacre** [comment]
Or move them to a database. 🙃

**Comment by chadwhitacre** [comment]
I'm good with this once the change you requested is made. 👍

**Comment by guergabo** [comment]
Hi @vladh, I pushed the changes. They should be live now. LMK if all good. Thanks!

**Comment by vladh** [comment]
@guergabo Err, sorry, I just noticed that your `averageNumberOfDevs` is `8086`. I'm assuming you don't have 8086 developers employed — did you mean 23?

**Comment by guergabo** [comment]
I made the changes. Should be live EOD.

**Comment by vladh** [comment]
Gotcha, thank you! :)

**Comment by guergabo** [comment]
Hi @vladh, this should be to go now. Thanks!

**Comment by vladh** [comment]
@guergabo Welcome on board! 🎈 

One small request: could you please change “The OSS pledge initiative” in your blog post to say “The Open Source Pledge initiative”? The name of the initiative is “Open Source Pledge” and the domain is “osspledge.com” — maybe we should have made the domain “opensourcepledge.com” to avoid confusion.

Thanks for your work on these small changes!

**Comment by guergabo** [comment]
> @guergabo Welcome on board! 🎈 https://osspledge.com/members/antithesis/
> 
> One small request: could you please change “The OSS pledge initiative” in your blog post to say “The Open Source Pledge initiative”? The name of the initiative is “Open Source Pledge” and the domain is “osspledge.com” — maybe we should have made the domain “opensourcepledge.com” to avoid confusion.
> 
> Thanks for your work on these small changes!

Hi @vladh, sorry for the delay. I was OOO for a bit. I made the change you requested. Should be live some time tomorrow. Thanks !

**Comment by vladh** [comment]
Thanks @guergabo! We've also changed the domain to opensourcepledge.com in the meantime to avoid confusion. :)

**Pull request merged**
